File: eighth/python/3.py
def somatorio_n_primeiros_inteiros(n):
    somatorio = sum(range(1, n+1))
    return somatorio

# Exemplo de uso
n = 5
resultado = somatorio_n_primeiros_inteiros(n)
print(f"O somatório dos {n} primeiros números inteiros é: {resultado}")

File: eighth/c/3.c
#include <stdio.h>

int somatorioNPrimeirosInteiros(int n) {
    int somatorio = 0;
    for (int i = 1; i <= n; i++) {
        somatorio += i;
    }
    return somatorio;
}

int main() {
    int n = 5;
    int resultado = somatorioNPrimeirosInteiros(n);
    printf("O somatório dos %d primeiros números inteiros é: %d\n", n, resultado);
    return 0;
}
